packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -21,26 +21,26 @@ class BotDetector {
   const BotDetector();
 
   bool get isRunningOnBot {
-    return
-      platform.environment['BOT'] == 'true' ||
+    return platform.environment['BOT'] != 'false'
+       && (platform.environment['BOT'] == 'true'
 
-          // https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables
-          platform.environment['TRAVIS'] == 'true' ||
-          platform.environment['CONTINUOUS_INTEGRATION'] == 'true' ||
-          platform.environment.containsKey('CI') || // Travis and AppVeyor
+           // https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables
```

Comment:
The indentation of the comments looks weird here

Reply:
It's lined up with the the expression to which they refer